lib/types/version-number.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
```

Comment:
This needs docs.  Also, should it be public?  Might be generally useful, but we could also consider hiding it..

Reply:
I don't think that, from a driver client perspective, it is necessary to make it public.

Reply:
I think its good as is for now, inside the `types` folder but not exported in the `types` module.
You could add `@ignore` to the jsdoc when documenting, if you want to keep it this way